src/Plug.vala

```diff
@@ -16,11 +16,8 @@
 //
 
 public class About.Plug : Switchboard.Plug {
-    private string kernel_version;
-    private string support_url;
-    private Gtk.Label based_off;
-
-    private string upstream_release;
+    private Gtk.Stack stack;
+    private OperatingSystemView operating_system_view;
```

Comment:
Is there a reason these are scoped to the plug class? They don't seem to be accessed outside of `get_widget ()`